__tests__/app.test.js

```diff
@@ -202,6 +202,45 @@ describe("/api/", () => {
     });
   });
 });
+describe.only("/api/reviews/:review_id/comments", () => {
+  test("200: responds with an array of comments associated with the passed id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const body = response.body.comments;
+        expect(body.length === 3).toBe(true);
+        body.forEach((comment) => {
+          expect(comment.hasOwnProperty('comment_id')).toEqual(true);
+          expect(comment.hasOwnProperty('body')).toEqual(true);
+          expect(comment.hasOwnProperty('review_id')).toEqual(true);
+          expect(comment.hasOwnProperty('author')).toEqual(true);
+          expect(comment.hasOwnProperty('votes')).toEqual(true);
+          expect(comment.hasOwnProperty('created_at')).toEqual(true);
+        });
```

Comment:
it would be worthwhile to be more specific when you can in your tests

we know every comment should have an `review_id` of `2` here!

Reply:
jset has a handy matcher that would help here

https://jestjs.io/docs/expect#tohavepropertykeypath-value

```js
expect(tutor).toHaveProperty("name", "doug")
```